wasm/src/types/scalar.rs

```diff
@@ -47,6 +47,11 @@ impl Scalar {
         self.0.to_string()
     }
 
+    /// Constructs a Scalar from its native representation that can be exposed to JavaScript.
+    pub(crate) fn from_native(native: ScalarNative) -> Self {
```

Comment:
This is already implemented in the From trait below. This method isn't really needed.